src/internal/behavior-pack/feature_rule.ts

```diff
@@ -0,0 +1,42 @@
+import { FormatVersion } from "../types/format-version";
+
+/** */
+export interface FeatureRule extends Readonly<FormatVersion> {
+    /** */
+    format_version: string;
+    /** */
+    "minecraft:feature_rules": {
+        description: {
+            identifier: string,
+            places_feature: string
+        },
+        conditions: {
+            placement_pass: string,
+            "minecraft:biome_filter": object
+        },
+        distribution: {
+            iterations: number | string,
+            x: object | string,
+            y: object | string,
+            z: object | string
```

Comment:
I rather see if we can santize the values to one or the other 🤔 makes working with it easier.

But if you say that its easier this way, I get it.

I do would love to see object being replaced with something more concrete, even if its just a Record<string, any>